cluster/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -296,6 +296,24 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
```

Comment:
@humblec @Madhu-1 In the rook patch releases, our goal is to make RBAC changes optional. What will happen if this RBAC is not applied? Will the snapshotter container just write errors to the log? Or will there be any other side effect?

Reply:
User cannot create and delete snapshots

Reply:
Yes snapshotter will keep writing error to logs